Statut HSWro.tex

```diff
@@ -41,54 +41,52 @@ \section{CZŁONKOWIE}
         \item zwyczajnych,
         \item honorowych.
       \end{enumerate}
-    \item Członkiem Zwyczajnym Stowarzyszenia może być osoba fizyczna, która spełnia stosowne postanowienia ustawy Prawo o Stowarzyszeniach oraz wszystkie poniższe warunki:
+    \item Członkiem Zwyczajnym Stowarzyszenia może być osoba fizyczna, która spełni wszystkie poniższe warunki:
       \begin{enumerate}
+         \item przedstawi pozytywną opinię co najmniej trzech członków zwyczajnego Stowarzyszenia,
         \item złoży deklarację członkowską na piśmie,
-        \item przedstawi pozytywną opinię co najmniej jednego członka zwyczajnego Stowarzyszenia
+        \item uzyska akceptację deklaracji członkowskiej uchwałą Zarządu Stowarzyszenia.
       \end{enumerate}
     \item Założyciele stają się członkami zwyczajnymi w~momencie zawiązania się Stowarzyszenia.
-    \item Osoba fizyczna staje się członkiem zwyczajnym stowarzyszenia po zaakceptowaniu pisemnej deklaracji członkowskiej uchwałą Zarządu Stowarzyszenia.
-    \item Członkiem Honorowym Stowarzyszenia może być osoba fizyczna, która spełnia stosowne postanowienia ustawy Prawo o Stowarzyszeniach oraz wszystkie poniższe warunki:
+    \item Członkiem Honorowym Stowarzyszenia może być osoba fizyczna, która spełni wszystkie poniższe warunki:
       \begin{enumerate}
-        \item jest pełnoletnia,
         \item wyrazi pisemną zgodę na nadanie jej w/w tytułu,
         \item wniosła wybitny wkład w działalność i rozwój Stowarzyszenia lub posiada znaczący wkład w dziedzinach zgodnych z celami Stowarzyszenia,
         \item zostanie zgłoszona przez co najmniej 10 członków zwyczajnych Stowarzyszenia lub Zarząd.
+        \item uzyska akceptacje zgłoszenia uchwałą Walnego Zebrania Członków
       \end{enumerate}
-    \item Osoba fizyczna staje się członkiem honorowym stowarzyszenia po zaakceptowaniu wniosku na Walnym Zebraniu Członków.
     \item Członkowie zwyczajni Stowarzyszenia mają prawo:
       \begin{enumerate}
         \item korzystania z dorobku, majątku i wszelkich form działalności Stowarzyszenia,
         \item udziału w zebraniach, wykładach oraz imprezach organizowanych przez Stowarzyszenie,
         \item zgłaszania wniosków dotyczących działalności Stowarzyszenia,
-        \item biernego i czynnego uczestniczenia w wyborach do władz Stowarzyszenia.
+        \item biernego i czynnego uczestniczenia w wyborach do organów Stowarzyszenia.
       \end{enumerate}
     \item Członkowie zwyczajni Stowarzyszenia mają obowiązek:
       \begin{enumerate}
         \item aktywnego udziału w działalności Stowarzyszenia i realizacji jego celów,
-        \item przestrzegania statutu i uchwał władz Stowarzyszenia,
+        \item przestrzegania statutu i uchwał organów Stowarzyszenia,
```

Comment:
czemu władze zostały zmienione na organy?

Reply:
moze by komisje rewizyjna tez ujac? Pytanie do autora @not7cd ?

Reply:
Komisja rewizyjna to nie władza. Zarząd to władza. Władze zawierają się w organach. @iwankgb umie w język polski.